testcase/deployment.go

```diff
@@ -42,7 +42,7 @@ func (t *Deployment) BeforeBackup(config Config) {
 		Expect(err).ToNot(HaveOccurred())
 		t.namespace = nsObject.Name
 
-		t.timeout = 10 * time.Minute
+		t.timeout = 20 * time.Minute
```

Comment:
Is 20 mins enough?

Reply:
Let's try it some releases, we can increase again if it is not enough.